### ISSUE TITLE
Is it possible to have a Handle, but disable dismiss by it?

### ISSUE THREAD

**Issue by pequi-roido** [the49ltd/The49.Maui.BottomSheet#102]
First and foremost, congrats for the library!

So, is it possible to have a sheet that is essentialy movable by the handle, but not dismissable by it? IsCancelable works, but disables any kind of movement

I experimented with recreating the sheet in the Dismissed event, but it's not the most ideal solution

Thanks in advance!

**Comment by vhugogarcia** [comment]
I don't think that is possible by design from each platform on iOS and Android. By reviewing the platforms documentation, I noticed there is a way to prevent to hide it, but not exactly as you wanted it. However, I may be wrong.
